scripts/test_updates_pg13.sh

```diff
@@ -10,7 +10,8 @@ source ${SCRIPT_DIR}/test_functions.inc
 run_tests "$@" -v7 \
           2.1.0-pg13 2.1.1-pg13 2.2.0-pg13 2.2.1-pg13 2.3.0-pg13 2.3.1-pg13 \
           2.4.0-pg13 2.4.1-pg13 2.4.2-pg13
+run_tests "$@" -r -v8 \
+          2.10.0-pg13 2.10.1-pg13
 run_tests "$@" -v8 \
           2.5.0-pg13 2.5.1-pg13 2.5.2-pg13 2.6.0-pg13 2.6.1-pg13 2.7.0-pg13 2.7.1-pg13 2.7.2-pg13 \
           2.8.0-pg13 2.8.1-pg13 2.9.0-pg13 2.9.1-pg13 2.9.2-pg13 2.9.3-pg13 2.10.0-pg13 2.10.1-pg13
```

Comment:
```suggestion
run_tests "$@" -v8 \
          2.5.0-pg13 2.5.1-pg13 2.5.2-pg13 2.6.0-pg13 2.6.1-pg13 2.7.0-pg13 2.7.1-pg13 2.7.2-pg13 \
          2.8.0-pg13 2.8.1-pg13 2.9.0-pg13 2.9.1-pg13 2.9.2-pg13 2.9.3-pg13
run_tests "$@" -r -v8 \
          2.10.0-pg13 2.10.1-pg13
```
Don't need to run twice tests for 2.10.x versions.